oadp-operator/controllers/restic.go

```diff
@@ -230,6 +230,12 @@ func (r *VeleroReconciler) ReconcileResticDaemonset(log logr.Logger) (bool, erro
 		},
 	}
 
+	if velero.Spec.EnableRestic != nil && !*velero.Spec.EnableRestic {
```

Comment:
@dymurray do we have the ability to see if restic is in use? I could see a case where a backup is running, we probably don't want to blindly delete this unlesss there is a finalizer that velero uses

Reply:
@shawn-hurley the only way to do this would be to loop through all PVBs and PVRs and check if any are in progress. You bring up a good point. I'm unaware of any finalizers that Velero uses to stop this from happening

Reply:
maybe just something to think about for the furture?